EuroPythonBot/registration/cog.py

```diff
@@ -27,7 +27,7 @@ async def callback(self, interaction: discord.Interaction) -> None:
         await interaction.response.send_modal(RegistrationForm(parent_cog=self.parent_cog))
 
 
-class RegistrationForm(discord.ui.Modal, title="Europython 2023 Registration"):
+class RegistrationForm(discord.ui.Modal, title="Europython Registration"):
```

Comment:
```suggestion
class RegistrationForm(discord.ui.Modal, title="EuroPython 2024 Registration"):
```

IMHO if it is publicly visible, it should include the year